requirements/quality.txt

```diff
@@ -183,7 +183,7 @@ pymongo==3.12.1
     # via
     #   -r requirements/test.txt
     #   edx-opaque-keys
-pyparsing==3.0.3
+pyparsing==2.4.7
```

Comment:
`packaging==21.2` requires `pyparsing<3` (see change https://github.com/pypa/packaging/pull/471). This constraint required will be fixed in the next version.